builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -124,12 +124,12 @@ local function start_install(package, reason)
 
 	local function callback(result)
 		if result.msg then
-			gamedata.errormessage = result.msg
+			gamedata.errormessage = fgettext("Error downloading $1 by $2: $3", package.title, package.name, result.msg)
```

Comment:
should the title be in quotes?
also I think the author can be left off and it will still be sufficiently clear